utils/dataset_validator.py

```diff
@@ -139,10 +139,10 @@ def validate_class(cls, idx=None, recordings_required=True):
     # Name
```

Comment:
We have some tests for the dataset validator at https://github.com/RashiSah/acousticbrainz-server/blob/b00f1f849ef2ba6041c308755c7de4dbcb90f699/utils/dataset_validator.py. It would be good to make sure that we have tests for these three methods that you changed so that we can be sure that they work as expected.

Reply:
@alastair 
Can you please explain me a bit more about the tests you are talking about here: https://github.com/RashiSah/acousticbrainz-server/blob/b00f1f849ef2ba6041c308755c7de4dbcb90f699/utils/dataset_validator.py ?

Reply:
Whoops, I meant to say https://github.com/RashiSah/acousticbrainz-server/blob/b00f1f849ef2ba6041c308755c7de4dbcb90f699/utils/test/test_dataset_validator.py

It looks like we have tests for most things regarding the structure of the dataset dictionary, but we don't have any tests to check that an error is shown if the names of classes or datasets are longer or shorter than the given values. We should add 3 more tests that provide names that are too short, and too long, and check that the correct exception is raised.

Reply:
Added tests for the dataset or class names that are too short or too long than the specified ranges.